### PR TITLE
Post - Drag and drop Lets Create

### PR DESCRIPTION
**Pull request by jiefmoreno**
# Description
Drag and drop du projet Lets create

### CODE REVIEW

**Comment by ElevenWilson** [comment]
31 words might be misspelled, please check them: clientX, clientY, cliqué, const, findIndex, func, getEquation, getObjectRatio, getObjectScale, getPosition, getPositionForElement, getResizeBack, getResizeBottom, getResizeGround, getResizeLeft, getResizeWrite, isInSection, isInZone, jiefmoreno, js, letscreate, mouseMoveEvent, mousePosition, objectHeight, objectId, offsetHeight, offsetLeft, offsetTop, offsetWidth, redimensionnement, trackpad

File: _drafts/2017-09-28-drag-and-drop-letscreate.md [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+
+### Problématique
+Dans cet article, je vais vous décrire la partie drag and drop du jeu. Je m'attarderais particulièrement sur le redimensionnement des objets.
+Je vous propose de réaliser une version simplifiée de cette fonctionnalité, en utilisant simplement javascript.
+
+### Le drag and Drop
+
+Tout d'abord, voici les images que nous allons utiliser:
+
+<div><img src="/assets/2017-09-28-drag-and-drop-letscreate/room.jpeg" style="width: 600px; margin: 10px;"/></div>
```
Comment (CaptainJojo) [inline _drafts/2017-09-28-drag-and-drop-letscreate.md]: utilise le markdown plz :)

Comment (VEBERArnaud) [reply _drafts/2017-09-28-drag-and-drop-letscreate.md]: il faut egalement que tu prefixe les url par le le `site.baseurl` 

```
![]({{ site.baseurl }}/assets/2017-09-28-drag-and-drop-letscreate/room.jpeg)
```

**Comment by ElevenWilson** [comment]
34 words might be misspelled, please check them: baseurl, clientX, clientY, cliqué, const, findIndex, func, getEquation, getObjectRatio, getObjectScale, getPosition, getPositionForElement, getResizeBack, getResizeBottom, getResizeGround, getResizeLeft, getResizeWrite, isInSection, isInZone, jiefmoreno, jpeg, js, letscreate, mouseMoveEvent, mousePosition, objectHeight, objectId, offsetHeight, offsetLeft, offsetTop, offsetWidth, png, redimensionnement, trackpad

**Comment by ElevenWilson** [comment]
34 words might be misspelled, please check them: baseurl, clientX, clientY, cliqué, const, findIndex, func, getEquation, getObjectRatio, getObjectScale, getPosition, getPositionForElement, getResizeBack, getResizeBottom, getResizeGround, getResizeLeft, getResizeWrite, isInSection, isInZone, jiefmoreno, jpeg, js, letscreate, mouseMoveEvent, mousePosition, objectHeight, objectId, offsetHeight, offsetLeft, offsetTop, offsetWidth, png, redimensionnement, trackpad

**Comment by ElevenWilson** [comment]
34 words might be misspelled, please check them: baseurl, clientX, clientY, cliqué, const, findIndex, func, getEquation, getObjectRatio, getObjectScale, getPosition, getPositionForElement, getResizeBack, getResizeBottom, getResizeGround, getResizeLeft, getResizeWrite, isInSection, isInZone, jiefmoreno, jpeg, js, letscreate, mouseMoveEvent, mousePosition, objectHeight, objectId, offsetHeight, offsetLeft, offsetTop, offsetWidth, png, redimensionnement, trackpad

**Comment by ElevenWilson** [comment]
34 words might be misspelled, please check them: baseurl, clientX, clientY, cliqué, const, findIndex, func, getEquation, getObjectRatio, getObjectScale, getPosition, getPositionForElement, getResizeBack, getResizeBottom, getResizeGround, getResizeLeft, getResizeWrite, isInSection, isInZone, jiefmoreno, jpeg, js, letscreate, mouseMoveEvent, mousePosition, objectHeight, objectId, offsetHeight, offsetLeft, offsetTop, offsetWidth, png, redimensionnement, trackpad

**Comment by ElevenWilson** [comment]
34 words might be misspelled, please check them: baseurl, clientX, clientY, cliqué, const, findIndex, func, getEquation, getObjectRatio, getObjectScale, getPosition, getPositionForElement, getResizeBack, getResizeBottom, getResizeGround, getResizeLeft, getResizeWrite, isInSection, isInZone, jiefmoreno, jpeg, js, letscreate, mouseMoveEvent, mousePosition, objectHeight, objectId, offsetHeight, offsetLeft, offsetTop, offsetWidth, png, redimensionnement, trackpad

**Comment by ElevenWilson** [comment]
35 words might be misspelled, please check them: baseurl, clientX, clientY, cliqué, const, findIndex, func, getEquation, getObjectRatio, getObjectScale, getPosition, getPositionForElement, getResizeBack, getResizeBottom, getResizeGround, getResizeLeft, getResizeWrite, ineteractive, isInSection, isInZone, jiefmoreno, jpeg, js, letscreate, mouseMoveEvent, mousePosition, objectHeight, objectId, offsetHeight, offsetLeft, offsetTop, offsetWidth, png, redimensionnement, trackpad

**Comment by ElevenWilson** [comment]
34 words might be misspelled, please check them: baseurl, clientX, clientY, cliqué, const, findIndex, func, getEquation, getObjectRatio, getObjectScale, getPosition, getPositionForElement, getResizeBack, getResizeBottom, getResizeGround, getResizeLeft, getResizeWrite, isInSection, isInZone, jiefmoreno, jpeg, js, letscreate, mouseMoveEvent, mousePosition, objectHeight, objectId, offsetHeight, offsetLeft, offsetTop, offsetWidth, png, redimensionnement, trackpad

**Pull request merged**